### PR TITLE
[#3095] Feature/3095 enable disable toggle control center

### LINKED ISSUE #3095: Enable/Disable toggle button on "Status" view
On Control center, Status view has enable/disable toggle button. 
We need to implement the functionality of enabling/disabling components from this view. 

1. Check endpoint list of installed components
2. Update component.status with the action of toggle button.
3. Show disabled component. 

FEATURE

We need to allow the user to disable services from status view on control center. 

Successful user story: 

1. User agent is on Status view.
2. User agent views a Component
3. User agent needs to disable a Component e.g:Chatplugin source
4. User agent clicks on the toggle button to disable a Component and its dependent services. 
5. A pop-up appear to confirm disabling a component. 
6. Component is shown on Status view as "toggle turned off" = disabled component. 
7. To Enable again, user is navigated to Component config. view. 

----------------------------------------------------------------------

FIGMA URL: https://www.figma.com/file/V3bEH3uxZ7ugePtndKgRmK/Control-Center-0.4?node-id=8%3A1018

### PR DESCRIPTION
**Pull request by juan-sebastian**
closes #3095

### CODE REVIEW

**Review by AudreyKj — COMMENTED** [review]
looking good :) 👍

File: frontend/control-center/src/pages/Status/ComponentListItem/ItemInfo.tsx [hunk 1]
```diff
@@ -1,27 +1,54 @@
...
   const isVisible = isExpanded || isComponent;
+  const {t} = useTranslation();
+
+  const triggerEnableDisableAction = (enabled: boolean) => {
+    enableDisableComponent({components: [{name: itemName, enabled: enabled}]});
+    setComponentEnabled(enabled);
+    setEnablePopupVisible(false);
+  };
+
+  const enableHandler = (enabled: boolean) => {
```
Comment (chrismatix) [inline frontend/control-center/src/pages/Status/ComponentListItem/ItemInfo.tsx:44]: Nitpick: The idiomatic way to name these two functions would be something more like `onEnableComponent` and `onToggleEnableComponent` ("Component" optional).

**Review by AitorAlgorta — APPROVED** [review]
Apart from the already existing comments. LGTM 🚀

**Review by AudreyKj — DISMISSED** [review]

File: lib/typescript/translations/translations.ts [hunk 2]
```diff
@@ -787,6 +791,8 @@ const resources = {
       searchTemplates: 'Recherche de modèles',
 
       //Control-Center
+      disableComponent: 'Désactiver',
+      disableComponentText: 'Tu es sûre de vouloir désactiver ce composant?',
```
Comment (AudreyKj) [inline lib/typescript/translations/translations.ts:795]: nitpick: peut-être vouvoiement?

`Êtes vous sûr(e) de vouloir désactiver ce composant ?`

Comment (juan-sebastian) [reply lib/typescript/translations/translations.ts:795]: I think is better to keep an informal tone. At least is what I see in many other apps. I also did an informal tone on Spanish and German

**Comment by juan-sebastian** [comment]
The cypress tests will be done in another PR

**Review by AitorAlgorta — APPROVED** [review]

**Pull request merged**